vector/src/main/java/im/vector/app/AutoRageShaker.kt

```diff
@@ -68,6 +68,7 @@ class AutoRageShaker @Inject constructor(
     fun initialize() {
         observeActiveSession()
         // It's a singleton...
+        enable(vectorPreferences.labsAutoReportUISI())
```

Comment:
Please do not insert new code between a line of code and its comment :)

Reply:
done